### ISSUE TITLE
multiple cursor support?

### ISSUE THREAD

**Issue by AlejandroNQ** [vinsworldcom/nppQuickText#24]
I'd love to make the npp equivalent to the vim
```
snippet beg "begin{} / end{}" bA
\begin{$1}
	$0
\end{$1}
endsnippet
```
note that the first hotspot doubles the cursor.

Is there any way of doing that? If not, it'd be a nice addition

**Comment by vinsworldcom** [comment]
Yes, I'd like that too!  The [DoxyIt](https://github.com/dail8859/DoxyIt/) plugin has an example of this.  I don't see it happening without a total code rewrite and as I didn't create this plugin, just maintain basic functionality so it isn't abandoned, I would welcome help in realizing these new features.

Cheers.
